### PR TITLE
add table for cron jobs

### PR DESCRIPTION
**Pull request by Nef10**
new version of table for upcoming cron jobs

### CODE REVIEW

**Comment by joshk** [comment]
maybe instead of having the hour, weekday and month columns, we could just have a `type` column (daily, weekly, and monthly could be values) and use the `created_at` timestamp?

**Comment by joshk** [comment]
@rkh thoughts?

**Comment by Nef10** [comment]
@joshk we changed it as requested

File: db/migrate/20160107120927_add_cron_table.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class AddCronTable < ActiveRecord::Migration
+
+  def up
+    execute "CREATE TYPE cron_type AS ENUM ('daily', 'weekly', 'monthly');"
+    create_table :crons do |t|
+      t.references :branch
+      t.column :type, :cron_type
```
Comment (joshk) [inline db/migrate/20160107120927_add_cron_table.rb]: sorry, I might have not explained myself, this should just be a string as there is no postgres type of `cron_type`

**Comment by jchromik** [comment]
@joshk we create this type with this line:
`execute "CREATE TYPE cron_type AS ENUM ('daily', 'weekly', 'monthly');"` (line 4)
On our local machines it worked perfectly.

**Comment by jchromik** [comment]
The CREATE TYPE statement is part of PostgreSQL since version 9.1 which was released in September 2011. If the PostgreSQL version you use is older than that, we have to switch to string.

cf. 
http://www.postgresql.org/docs/9.1/static/sql-createtype.html and
https://en.wikipedia.org/wiki/PostgreSQL#Release_history

**Comment by joshk** [comment]
Sorry, completely missed line 4.

Postgres versions are fine as we use 9.3. I wonder how well this will work with ActiveRecord though.

@rkh @drogus what are your thoughts on us using ENUM types?

**Comment by rkh** [comment]
Hm. Dunno, we usually just do strings. Either way, the `down` method should also remove the type again, shouldn't it?

**Comment by jchromik** [comment]
Oh, of course. Changing the down method appropriately will be the first thing we'll do on Monday (given that we may use the ENUM type instead of string).

**Comment by joshk** [comment]
seems good to me, the @rkh seal of approval is needed. 

also, the tests are failing, not sure why, but if you can get it to pass that would be great

**Comment by Nef10** [comment]
@joshk could you restart the failed build? We think this is not related to our changes as this happened [before](https://travis-ci.org/travis-ci/travis-core/jobs/93142477)

**Comment by rkh** [comment]
Looks good. I've restarted the build.

**Comment by rkh** [comment]
Can you pull in the latest changes from master?

**Comment by Nef10** [comment]
Looks like after this merge one job needs another restart

**Comment by Nef10** [comment]
now opened on migration repository travis-ci/travis-migrations#6

**Pull request closed without merging**